### ISSUE TITLE
vlc captures entire taskbar thumnail and wont let other widget present in window to show.

### ISSUE THREAD

**Issue by hacker6914** [oaubert/python-vlc#161]
I am embedding python vlc in pyqt5 to make a video player.I never noticed whats going on taskbar,suddenly i came to know that  when video starts to play it takes entire taskbar thumbnail to show only its video not other items present in the window screen.And after closing the video the taskbar becomes totally blank white, not showing anything.Is there any method to in the python-vlc to restore it normal.
i have posted my every details with images for better understanding.in the stackoverflow [here](https://stackoverflow.com/questions/65061620).But wont get any desired answer.Please check.

**Comment by mrJean1** [comment]
This is almost certainly a PyQt5 or Qt (or Windows) issue.  Neither `python-vlc` nor `libVLC` tinker with windows, banners or any other UI widgets.  The `vlc.player` is passed a handle to the window to supply the contents, that is all.  Any window decorations, buttons, menus, etc. are created outside and without involvement of `python-vlc` and `libVLC`.

Take a closer look at the `pyqt5vlc.py` example and/or try to run that in your particular environment.  If that does not help, check the PyQt5 or Qt sites.

**Comment by hacker6914** [comment]
No,this is a vlc issue,cause vlc set forcely only its video rendering area into taskbar.by using windows API "SetThumbNailClip" which is used to set a particular area of application into taskbar.
![Screenshot (12)](https://user-images.githubusercontent.com/67958180/112754718-35783680-8f92-11eb-8478-d0815a9c5e69.png)

I raised and solved this issue at stackoverflow. Please look at the link [](https://stackoverflow.com/questions/65146103/how-to-select-a-particlular-portion-of-a-windows-client-area-to-display-in-wind)

**Comment by AppyxDaniel** [comment]
I experience the exact same behavior using LibVlc.WPF. 
LibVlc produces the same log entry: 

`direct3d11 Error: SetThumbNailClip failed: 0x800706f4`

**Comment by mrJean1** [comment]
If the link above doesn't work, [this one does](https://stackoverflow.com/questions/65146103/).

**Comment by AppyxDaniel** [comment]
I was able to reset the clip region of the thumbnail using the same Win32 API as LibVLC does: https://learn.microsoft.com/en-us/windows/win32/api/shobjidl_core/nf-shobjidl_core-itaskbarlist3-setthumbnailclip

For anyone interested where the thumbnail clipping happens:
https://github.com/videolan/vlc/blob/3.0.x/modules/video_output/win32/common.c
See method `CommonChangeThumbnailClip`
